drupal/sites/all/features/boinc_standard/boinc_standard.features.menu_links.inc

```diff
@@ -82,11 +82,10 @@ function boinc_standard_menu_default_menu_links() {
   );
   // Translatables
   // Included for use with string extractors like potx.
```

Comment:
This comment (also present in lots of other places) is inaccurate. As we found out the `t()` or `bts()` function needs to be called at least once for a non-english language so that the string gets added to the Drupal Translation Interface (DTI). This means that the project that is providing the canonical translation templates for Drupal needs to visit each webpage on its site to ensure that every string is added to the DTI and thus is part of the exported template. The comment suggests that some static string extractor is used which is not the case for the "boinc" and "project" textgroups currently. For projects importing translations this is not a problem as those get added to the DTI automatically.